### DISCUSSION TITLE
Comment Restructure

### DISCUSSION THREAD

**Discussion by iSupremacy** [civitai/civitai#308] [Ideas]
I enjoy the site a lot, but the Comments are just thrown in with reviews. Should try to give them their own section and how they look.

**Comment by WASasquatch** [comment]
It would be cool if it was tabbed. Or do what a lot of other sites do and reserve the footer for comments/discussion and have the summarized reviews (latest) on a left/right hand panel, where you can then go and view all reviews. 

Additionally; it would be cool if both comments and reviews can post images, and those images go into a "Community Images" section of a model where you could just view all the images people make with it.
